test/test.geocoder.js

```diff
@@ -109,6 +109,77 @@ test('geocoder', function(tt) {
     );
   });
 
+  tt.test('options.reverseGeocode - true by default', function(t) {
+    t.plan(3);
+    setup();
+    geocoder.query('34.5177548, -6.1933875');
+    geocoder.on(
+      'results',
+      once(function(e) {
+        t.equal(e.features.length, 1, 'One result returned');
+        t.equal(
+          e.features[0].place_name,
+          'Singida, Tanzania',
+          'returns expected result'
+        );
+        t.equal(e.config.limit, 1, 'sets limit to 1 for reverse geocodes');
+      })
+    );
+  });
+
+  tt.test('options.reverseGeocode - false', function(t) {
+    t.plan(2);
+    setup({
+      reverseGeocode: false
+    });
+
+    geocoder.query('34.5177548, -6.1933875');
+    geocoder.on(
+      'results',
+      once(function(e) {
+        t.equal(e.features.length, 0, 'No results returned');
+      })
+    );
+    geocoder.on(
+      'error',
+      once(function(e) {
+        t.equal(e.error.statusCode, 422, 'should error');
+      })
+    );
+  });
+
+  tt.test('parses options correctly', function(t) {
+    t.plan(4);
+    setup({
+      language: 'en, es, zh',
+      types: 'district, locality, neighborhood, postcode',
+      countries: 'us, mx'
```

Comment:
we should add a unit test for non spaces. eg. `'district,locality'` to ensure this is still supported and backwards compatible

Reply:
Done 👍